cache-config/t3c-apply/config/config.go

```diff
@@ -100,6 +100,11 @@ type Cfg struct {
 	DisableParentConfigComments bool
 	DefaultClientEnableH2       *bool
 	DefaultClientTLSVersions    *string
+	// MaxMindLocation is a URL string for a download location for a maxmind database
+	// for use with either HeaderRewrite or Maxmind_ACL plugins
+	MaxMindLocation string
+	TsHome          string
```

Comment:
Why is this PR adding `TsHome` and `TsConfigDir`? Merge error?

Reply:
We need the configdir to know where to store the mmdb, home I just brought a long for completeness in the data structure if it was ever needed

Reply:
Ah, I missed the usage, I see it now